packages/@okta/vuepress-site/docs/guides/password-import-hook/dependencies/nodejs/dependencies.md

```diff
@@ -8,7 +7,7 @@ const { body, validationResult } = require('express-validator');
 const bodyParser = require('body-parser');
 ```
 
-We're also going to separate out into its own file the code that implements the business logic of this external service, which is the actual check of the submitted user credentials against our user store:
+Our implementation of the external service also separates into its own file module the code that implements the business logic of our external service, which is the actual check of the submitted user credentials against our user store. The file module containing this code is named `users`:
```

Comment:
I feel like commas would make this more clear. I had to read it a couple of times to discover where the "pause" was so that I understood:

Our implementation of the external service also separates, into its own file module, the code that implements the business logic of our external service.....

also last sentence: The file module **that contains** this code (for translation)